### PR TITLE
added - for amalgamation month and year

### PR DESCRIPTION
**Pull request by JaswiniP**

### CODE REVIEW

**Review by RogerHowellDfE — APPROVED** [review]

File: Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml [hunk 1]
```diff
@@ -266,14 +266,14 @@
                             </div>
                             <div class="govuk-date-input__item">
                                 <div class="govuk-form-group">
-                                    <label for="amalgamationdate-month" class="govuk-label govuk-date-input__label">Month</label>
-                                    <input class="govuk-input date-text-month" type="text" v-model="mergeDateMonth" id="amalgamationdate-month">
+                                    <label for="amalgamation-date-month" class="govuk-label govuk-date-input__label">Month</label>
```
Comment (antk43) [inline Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml:269]: @JaswiniP should the for value be changed here? FYI @RogerHowellDfE

File: Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml [hunk 2]
```diff
@@ -266,14 +266,14 @@
...
+                                    <input class="govuk-input date-text-month" type="text" v-model="mergeDateMonth" id="amalgamation-date-month">
                                 </div>
                             </div>
                             <div class="govuk-date-input__item">
                             <div class="govuk-form-group">
-                                <label for="amalgamationdate-year" class="govuk-label govuk-date-input__label">Year</label>
-                                    <input class="govuk-input gias-year-input__item date-text-year" type="text" v-model="mergeDateYear" id="amalgamationdate-year">
+                                <label for="amalgamation-date-year" class="govuk-label govuk-date-input__label">Year</label>
```
Comment (antk43) [inline Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml:275]: @JaswiniP should the for value be changed here? FYI @RogerHowellDfE

File: Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml (hunk 1 above)
Comment (ScottDawsonDfE) [reply Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml:269]: My best understanding - this value needs to match the model class

File: Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml (hunk 2 above)
Comment (ScottDawsonDfE) [reply Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml:275]: My best understanding - this value needs to match the model class

File: Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml (hunk 1 above)
Comment (RogerHowellDfE) [reply Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml:269]: The `for` attribute of a `<label>` needs to match the `id` value of the `<input>` element the label is describing - updating one needs the other to be updated also, so updating both here is fine :)

File: Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml (hunk 2 above)
Comment (RogerHowellDfE) [reply Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml:275]: See comment on the other one :)

**Pull request merged**